pkg/pillar/hypervisor/kvm.go

```diff
@@ -721,15 +722,33 @@ func (ctx KvmContext) Setup(status types.DomainStatus, config types.DomainConfig
 	return nil
 }
 
-// CreateDomConfig creates a domain config (a qemu config file, typically named something like xen-%d.cfg)
-func (ctx KvmContext) CreateDomConfig(domainName string, config types.DomainConfig, status types.DomainStatus,
-	diskStatusList []types.DiskStatus, aa *types.AssignableAdapters, file *os.File) error {
+// Coalesce per-app `EnableVncShimVM` flag and global `VncShimVmAccess'
+// debug flag, making sure we don't activate VNC for shim VM if VNC for
+// this application is disabled.
+func isVncShimVmEnabled(
+	globalConfig *types.ConfigItemValueMap, config types.DomainConfig) bool {
+	globalShimVnc := false
+	if globalConfig != nil {
+		item, ok := globalConfig.GlobalSettings[types.VncShimVmAccess]
+		globalShimVnc = ok && item.BoolValue
+	}
+	return config.EnableVnc && (config.EnableVncShimVM || globalShimVnc)
+}
+
+// CreateDomConfig creates a domain config (a qemu config file,
+// typically named something like xen-%d.cfg)
+func (ctx KvmContext) CreateDomConfig(domainName string,
```

Comment:
Please add the nocyclo comment here to shut off the yetus complaint.

Reply:
Super strange, as I understand cyclop and gocyclo are disabled in pillar/.golangci.yml. Will add then a comment

Reply:
@europaul helped me to figure out this is all about the 'maintidx', I disabled it globally for the whole pillar.